cmd/singularity/actions_test.go

```diff
@@ -126,6 +131,21 @@ func testSingularityRun(t *testing.T) {
 
 // testSingularityExec tests min fuctionality for singularity exec
 func testSingularityExec(t *testing.T) {
+	// Create a temp testfile
+	tmpfile, err := ioutil.TempFile("testdata", "testfile.tmp")
+	if err != nil {
+		t.Fatal(err)
+	}
+	testfile, err := tmpfile.Stat()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
It might be better to move this `pwd` assignment to just before you decide to change directories so its more obvious what the previous directory was. It doesn't appear to be used before in the test definitions?

Reply:
I am using the `pwd` var on multiple tests, check the tests list